TestFoundation/TestNSString.swift

```diff
@@ -227,6 +228,18 @@ class TestNSString : XCTestCase {
         let string = NSString(CString: bytes.map { Int8(bitPattern: $0) }, encoding: NSUTF8StringEncoding)
         XCTAssertNil(string)
     }
+    
+    func test_FromContentOfFile() {
+        let testFilePath = testBundle().pathForResource("NSStingTestData", ofType: "txt")
```

Comment:
There appears to be a typo in the path name here; shouldn't it be `NSStringTestData`? Does this test pass when you run the test suite?